lib/content-services/src/lib/tag/tag-node-list.component.ts

```diff
@@ -42,7 +42,20 @@ export class TagNodeListComponent implements OnChanges, OnDestroy, OnInit {
     @Input()
     showDelete = true;
 
-    tagsEntries: TagEntry[];
+    /** Should limit number of tags displayed */
+    @Input()
+    limitTagsDisplayed = false;
+
+    @ViewChild('nodeListContainer')
+    containerView: ElementRef;
+
+    @ViewChildren(MatChip)
+    tagChips: QueryList<MatChip>;
+
+    tagsEntries: TagEntry[] = [];
+    calculationsDone = false;
```

Comment:
can we specify data type for this variable.

Reply:
guys, usually we don't need explicit types in cases we provide the value, the linters even suggest removing types for Booleans and Strings (like you can see in many other places)